dask/dataframe/tests/test_rolling.py

```diff
@@ -270,7 +270,10 @@ def test_time_rolling_constructor():
 @pytest.mark.parametrize("window", ["1S", "2S", "3S", pd.offsets.Second(5)])
 def test_time_rolling_methods(method, args, window, check_less_precise):
     if dd._compat.PANDAS_GT_110:
-        check_less_precise = {}
+        if check_less_precise:
+            check_less_precise = {"atol": 0.5e-3, "rtol": 0.5e-3}
+        else:
+            check_less_precise = {"atol": 0.5e-5, "rtol": 0.5e-5}
```

Comment:
I think this can be removed? Or replaced with `check_less_precise = {}` probably.

We only want to set these higher tolerances when `check_less_precise` is True.

Reply:
Oh sorry I misread. Your changes are fine.

Reply:
ok that makes sense.

Reply:
those are slightly higher than default, but should be fine